blog/2022-02-24-starboard.md

```diff
@@ -0,0 +1,92 @@
+
+Great now we have metrics that we can trust, but wasn't the whole point of this work to continuously scan for new CVEs in the cluster?
+
+The problem was that once a vulnerability report got generated it didn't get updated unless the current vulnerability report was deleted and this creates issues for long-running deployments.
+To solve this we introduce a TTL(Time To Live) for vulnerability reports [#879](https://github.com/aquasecurity/starboard/pull/879) by implementing a new controller.
+The controller currently only supports managing TTL for vulnerability reports but could easily add the same feature to other Starboard reports.
+Setting the following config in the operator `OPERATOR_VULNERABILITY_SCANNER_REPORT_TT=24h` will automatically delete any vulnerability report older then 24 hours.
+
+So now we can scan our public images, we can get metrics from them and we can get updated scans as often as we want. But what about privates repositories?
+
+Support in Trivy for AWS [ECR](https://aws.amazon.com/ecr/) (Elastic Container Registry) have been around for a long time and same thing with starboard.
+The problem was until recently there was no support for Azure [ACR](https://azure.microsoft.com/en-us/services/container-registry) (Azure Container Registry).
+
+Luckily just as we were thinking of starting to work on this feature someone else from the community did the heavy [lifting](https://github.com/aquasecurity/fanal/pull/371) and added support for Azure in Fanal.
+Fanal is the library that Trivy uses to scan images, and by fixing this together with a number of other commits to both Starboard and Trivy the problem was solved.
+
+Or so we thought...
+
+There had been a PR to add to the possibility of setting a custom label on your Starboard [jobs](https://github.com/aquasecurity/starboard/pull/902) thus giving us the possibility to of using [aad-pod-identity](https://github.com/Azure/aad-pod-identity) per Starboard job.
+Aad-pod-identity makes it possible to talk to Azure from a container without having to worry about passwords so it's something we definitely want to use.
+But instead of running Starboards vulnerability scan jobs in server mode (which is the default), we are running Starboard in client mode.
+We have deployed a separate Trivy instance in our cluster to cache all the images that we scan and this saves us allot of time per image that is used in the cluster.
+
+The problem was that the Trivy helm chart didn't support setting custom labels on the Trivy stateful set so we created a [PR](https://github.com/aquasecurity/trivy/pull/1767) to solve it.
+
+Thanks to this we are now able to quickly generate dashboards with the amount of critical CVEs in our clusters for both public and private images.
```

Comment:
In case you're fixing the tense of above paras, here present tense is correct.

Reply:
It would be good to say how/when this will be available to our customers (i.e. show up in their reports).